parmed/openmm/parameters.py

```diff
@@ -239,7 +239,8 @@ def from_parameterset(cls, params, copy=False, remediate_residues=True):
         return new_params
 
     @needs_lxml
-    def write(self, dest, provenance=None, write_unused=True, separate_ljforce=False,
+    def write(self, dest, provenance=None, write_unused=True, skip_duplicates=True,
+              separate_ljforce=False,
```

Comment:
Put `skip_duplicates` at the end to avoid breaking backwards compatibility.

Any arguments (except those after `*`) can be either keyword args or positional args.  If a script uses positional args, adding new arguments in the middle of the call signature will change the behavior of that function call.

That's why it's always best practice to add them at the end (unless you specify keyword-only arguments).

Reply:
See new commit (sorry if there's a better way of doing this - I'm still a bit fresh when it comes to working with GitHub).